approved/ultraflex/subroutines.atp

```diff
@@ -815,163 +814,94 @@ pipe_minus 1                                                     > tp0
+//  Using counter c15 as set_code replacement - value set to 100 + 1
+set_msb 1                                                                                       
+                                                                 > tp0                          0 0 X 0 ;
+set c15 101                                                      > tp0                          0 0 X 0 ;
 branch_expr = (cpuA_cond)                                        > tp0                          0 0 X 0 ;
 set_cpu_cond (cpuA_cond)                                         > tp0                          0 0 X 0 ;
 loop_here_8: if (branch_expr) jump loop_here_8                   > tp0                          0 0 X 0 ;
 repeat 2                                                         > tp0                          0 0 X 0 ;
 return                                                           > tp0                          0 0 X 0 ;
 global subr digsrc_overlay_testme32:                                                            
-((tdi,tms):DigSrc = Start)                                                                      
-repeat 145                                                       > tp0                          0 0 X 0 ;
-((tdi,tms):DigSrc = SEND)                                                                       
-                                                                 > tp0                          0 D X D ;
-((tdi,tms):DigSrc = SEND)                                                                       
-                                                                 > tp0                          0 D X D ;
-((tdi,tms):DigSrc = SEND)                                                                       
-                                                                 > tp0                          0 D X D ;
-((tdi,tms):DigSrc = SEND)                                                                       
-                                                                 > tp0                          0 D X D ;
-((tdi,tms):DigSrc = SEND)                                                                       
-                                                                 > tp0                          0 D X D ;
-((tdi,tms):DigSrc = SEND)                                                                       
-                                                                 > tp0                          0 D X D ;
-((tdi,tms):DigSrc = SEND)                                                                       
-                                                                 > tp0                          0 D X D ;
-((tdi,tms):DigSrc = SEND)                                                                       
+((tdi,tms):DigSrc = Start )                                                                     
+repeat 145                                                       > tp0                          0 D X D ;
```

Comment:
@ekingham we need to look at the implementation here. The vectors appear different (0 0 X 0 vs 0 D X D) which explains the one missing "Send" on the dig source.